app/src/main/java/com/example/songme/ui/playmusic/PageTrackCoverFragment.kt

```diff
@@ -5,13 +5,39 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import com.bumptech.glide.Glide
 import com.example.songme.R
+import com.example.songme.utils.Constants.IMAGE_KEY
+import kotlinx.android.synthetic.main.fragment_page_track_cover.*
 
 class PageTrackCoverFragment : Fragment() {
 
+    private var imageUrl: String? = null
+
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? =
-        inflater.inflate(R.layout.fragment_page_track_cover, container, false)
+    ): View? {
+        imageUrl = arguments?.getString(IMAGE_KEY)
+        return inflater.inflate(R.layout.fragment_page_track_cover, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        Glide.with(view)
+            .load(imageUrl)
+            .circleCrop()
+            .error(R.drawable.ic_music_black)
+            .into(imageTrackCover)
```

Comment:
Glide lặp lại nhiều lần thì ông viết Extension cho ImageView để load ảnh đi 
```suggestion
        imageTrackCover.load(imageUrl)
```